web/beacon-app/src/features/projects/types/projectService.ts

```diff
@@ -13,3 +19,7 @@ export type ProjectDetailQuery = {
 };
 
 export type ProjectDetailDTO = Pick<ProjectResponse, 'id'>;
+
+export interface ProjectQuery extends ProjectDetailQuery {
+  getProject(): void
+}
```

Comment:
I'm not sure it's a good idea to extend that. could you replace `project` with `projects` 
```suggestion
export interface ProjectsQuery {
  getProjecst(): void,
  hasProjectsFailed: query.isError,
    isFetchingProjects: query.isLoading,
    projecst: query.data,
    wasProjectsFetched: query.isSuccess,
    error: query.error,
}
```